core/management/commands/load_copyright.py

```diff
@@ -8,18 +8,17 @@
 from core.load_copyright import loadCopyright
 from core.management.commands import configure_logging
 
-configure_logging("load_copyright_logging.config", "load_coyright.log")
+configure_logging("load_copyright_logging.config", "load_copyright.log")
 
 LOGGER = logging.getLogger(__name__)
 
 
 class Command(BaseCommand):
-    help = "Load copyrights"
+    help = "Add copyright uris and labels to the Copyright table from input file."
```

Comment:
Same as the other command - long help would be awesome if django has a way to do that for commands.